resource/abstractions/auditing_rules.md

```diff
@@ -0,0 +1,478 @@
+---
+layout: default
+grand_parent: General Resource Admin
+parent: Abstractions
+---
+
+# Auditing Rules
+
+General Resource Profile Administration functions for manipulating a General Resource Profile's Auditing Rules. 
+{: .fs-6 .fw-300 }
+
+## `ResourceAdmin.get_audit_rules()`
+
+```python
+def get_audit_rules(self, resource: str, class_name: str) -> Union[dict, bytes, None]:
+```
+
+#### 📄 Description
+
+Check the **Audit Rules** for a general resource profile.
+
+#### 📥 Parameters
+* `resource`<br>
+  The **general resource profile** for which RACF should check the audit rules.
+
+* `class_name`<br>
+  The name of the **class** the general resource profile being checked belongs to.
+
+#### 📤 Returns
+* `Union[dict, bytes, None]`<br>
+  Returns `None` when the general resource profile has no **Auditing Rules** defined, otherwise returns the auditing rules as a dictionary. If the `ResourceAdmin.generate_requests_only` class attribute is set to `True`, **concatenated Security Request XML bytes** will be returned.
```

Comment:
```markdown
Returns `None` when the general resource profile has no **Auditing Rules** defined, otherwise returns the **Auditing Rules** as a dictionary. If the `ResourceAdmin.generate_requests_only` class attribute is set to `True`, **concatenated Security Request XML bytes** will be returned.
```

Reply:
Changed by above changes.

Reply:
Change `otherwise returns the **Auditing Rules** as a dictionary.` to `otherwise the **Auditing Rules** are returned as a dictionary.`

Reply:
changed.